sphinxcontrib/chapeldomain.py

```diff
@@ -508,14 +508,14 @@ class ChapelModuleLevel(ChapelObject):
     @property
     def chpl_type_name(self):
         """Returns iterator or procedure or '' depending on object type."""
-        if not self.objtype.endswith('function'):
+        if self.objtype == 'type':
+            return 'type'
+        elif not self.objtype.endswith('function'):
```

Comment:
It was causing problems...

The function is being conservative about objtype support, so it used to check `if not self.objtype.endswith('function')` first and return an empty string for that. type was falling into that branch and never getting to the one that returned 'type'.